src/engine/resolvers.rs

```diff
@@ -80,6 +81,65 @@ where
         }
     }
 
+    /// Returns the resolver input deserialized into a structure of type T that
+    /// implements the serde `Deserialize` trait.
+    ///
+    /// # Errors
+    ///
+    /// Returns an [`Error]` variant [`InputNotFound`] if no input field was passed
+    /// to the query, [`TypeConversionFailed`] if unable to convert a warpgrapher Value
```

Comment:
I'd recommend replacing "warpgrapher Value" "[`warpgrapher::Value`]" or whatever the linked fully qualified name would be.

Reply:
Fixed.